code/modules/mob/living/simple_animal/hostile/hog.dm

```diff
@@ -365,3 +365,89 @@ if ungreased adult: l containers
 
 /mob/living/simple_animal/hostile/spacehog/piglet/set_mood(var/mood)
 	..(HOG_SKITTISH)
+
+
+//This is not a subtype of spacehog because it is very different than other spacehogs. It does not fight, it has no moods, foraging, hunger needs, etc.
+/mob/living/simple_animal/rampagingspacehog
+	name = "rampaging overgreased feral space hog"
+	desc = "It leaves a sickly trail of grease and knocks over anyone in its way."
+	speed = 1 //moves at same speed as a person when not dashing
+	maxHealth = 450
+	health = 450
+	icon = 'icons/mob/hog.dmi'
+	icon_state = "hog_overgreased"
+	icon_living = "hog_overgreased"
+	icon_dead = "hog_overgreased_dead"
+	speak = list("Oink!","Squee!","Sqwaa!","Ounch!", "SQUEEEEE!","Oink...","Oink, oink", "Oink, oink, oink", "Oink!", "Oiiink.")
+	emote_hear = list("squeals hauntingly")
+	emote_see = list("roots about","squeals hauntingly")
+	emote_sound = list("sound/voice/pigsnort.ogg","sound/voice/pigsqueal.ogg")
+	response_help = "pats"
+	response_disarm = "shoves"
+	response_harm = "hits"
+	meat_type = /obj/item/weapon/reagent_containers/food/snacks/meat/box/pig
```

Comment:
by default, meat_amount gets set to the mob size; in this case, SIZE_BIG should be 4

however, since the meat is kind of the reward for the event (which is onetime and somewhat annoying), it might be worth overriding to give extra